### PR TITLE
support aliyun container service monitor

### PR DESCRIPTION
**Pull request by Yaiba**
Add dummy way to support aliyun container service.

### CODE REVIEW

File: dummy_support_aliyun_container.sh [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+	memcached_config=/etc/oneapm-ci-agent/conf.d/mcache.yaml
+	memcached_addr=`echo $MEMCACHED_PORT | cut -d : -f 2`
+	memcached_addr=${memcached_addr:2}
+	memcached_port=`echo $MEMCACHED_PORT | cut -d : -f 3`
+
+	cp $memcached_config.example $memcached_config
+	# host
+	sed -i -e "s/^  - url: localhost/  - url: ${memcached_addr}/" $memcached_config
+	# port
+  sed -i -e "s/^  #   port: 11211/    port: 11211/" $memcached_config
```
Comment (startover) [inline dummy_support_aliyun_container.sh]: pay attention to line indent.

File: Dockerfile [hunk 2]
```diff
@@ -30,6 +30,10 @@ RUN mv /etc/oneapm-ci-agent/oneapm-ci-agent.conf.example /etc/oneapm-ci-agent/on
 COPY conf.d/docker_daemon.yaml /etc/oneapm-ci-agent/conf.d/docker_daemon.yaml
 
 COPY entrypoint.sh /entrypoint.sh
+RUN chmod +x /entrypoint.sh
```
Comment (startover) [inline Dockerfile:33]: This is unnecessary.

Comment (Yaiba) [inline Dockerfile:33]: To ensure it truely works,  we better keep the command.

File: dummy_support_aliyun_container.sh (hunk 1 above)
Comment (Yaiba) [inline dummy_support_aliyun_container.sh]: indent has been fixed.

File: Dockerfile (hunk 2 above)
Comment (startover) [inline Dockerfile:33]: OK.

**Pull request merged**